.nvmrc

```diff
@@ -0,0 +1 @@
+8.9.2
```

Comment:
this can probably just be `8`.

Also, the file requires a trailing newline (like most unix files), or else `nvm` won't work properly.

Reply:
Updated, @ljharb!